config/config.go

```diff
@@ -180,6 +181,12 @@ func (t *GDPRTimeouts) ActiveTimeout() time.Duration {
 	return time.Duration(t.ActiveVendorlistFetch) * time.Millisecond
 }
 
+// TCF1 defines the TCF1 specific configurations for GDPR
+type TCF1 struct {
+	FetchGVL       bool   `mapstructure:"fetch_gvl"`
+	DefaultGVLPath string `mapstructure:"default_gvl_path"`
```

Comment:
The word "Default" in a config file leads me to think this might be, well, a default value and not one necessarily provided by the host. Consider renaming to `FallbackGVLPath`.